src/python_extension.ts

```diff
@@ -75,7 +75,7 @@ export class PythonExtension {
 		}
 		const editor = window.activeTextEditor;
 		this.inmantaEnvSelector.text = text;
-		if (editor && editor.document.languageId === "inmanta") {
+		if (editor && ['inmanta','log','pip-requirements','properties'].includes(editor.document.languageId)) {
```

Comment:
This is unrelated, right? Does it still only show when you're working on an inmanta module/project, or does it now show for any project's `requirements.txt`?

Reply:
Indeed, I noticed I forgot that in my previous pr to: 
It will now show when the active window is a .cf, vscode logs, a pip-requirements file or a config window (like the walk through).

Reply:
But still only if the active directory or one of the directories in the active workspace is an inmanta project/module? Or always for those file types, even for a plain Python project?

Reply:
First the extension needs to start ( currently you will still need to open a .cf file or to execute a command defined by the extension). After that our own env selector will start to show for those type of files.